{"name": "Проверка появления заказа в разделе 'В работе' после его оформления", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'setup' for <Function test_order_details_popup[firefox]>>\n\n    @pytest.fixture(params=[\"chrome\", \"firefox\"], scope=\"class\")\n    def setup(request):\n>       driver = initialize_driver(request.param)\n\nconftest.py:6: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nbrowser_name = 'firefox'\n\n    def initialize_driver(browser_name):\n        if browser_name == \"chrome\":\n            driver = webdriver.Chrome(ChromeDriverManager().install())\n        elif browser_name == \"firefox\":\n>           driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\nE           TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nutils\\driver_setup.py:9: TypeError"}, "parameters": [{"name": "setup", "value": "'firefox'"}], "start": 1730908181286, "stop": 1730908181286, "uuid": "1b2b904b-49eb-41b7-89f7-6050d6fedcc1", "historyId": "7b77a6f6e222c1612f05d228491e3cb9", "testCaseId": "25ec94c120032a7e8cd4671ad4155bab", "fullName": "tests.test_order_feed.TestOrderFeed#test_order_appears_in_progress_section", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "WorkPC"}, {"name": "thread", "value": "2056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}